### DISCUSSION TITLE
Maaf kak hanya numpang tanya ,gmana cara membuka web yg di blokir cloudflare

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1417] [Q&A] [answered]
![20230619_162655_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/a0998af5-263b-4638-b997-2f0a26c585de)

**Comment by realodix** [comment, marked as answer]
https://bokepav.com/ ya?

Cloudfalre menunjukkan info
- Browser working, artinya jaringan Anda tidak ada masalah
- Cloudflare working, artinya jaringan cloudflare juga tidak ada masalah
- Host error, artinya memang https://bokepav.com/ sedang down servernya

Saya coba cek di https://downforeveryoneorjustme.com/, hasilnya sama, saat ini https://bokepav.com/ sedang down
![nonameaaa](https://github.com/realodix/AdBlockID/assets/1314456/0718293f-329d-49fb-b0ec-4a07ddcf1bad)

Saat ini tidak ada yang dapat Anda lakukan selain menunggu adminnya untuk memulihkan websitenya tersebut.

Semoga jawaban Saya dapat membantu

**Reply by Jokopentil** [reply]
Ooo begitu ya saya sering menjumpai seperti itu 
Karena penasaran coba tanya
Trimakasi jawabannya

**Reply by Jokopentil** [reply]
Bahkbhkan saya pake vpn dan dns juga gk kebuka🤔🤔
